.packit.yaml

```
@@ -87,6 +87,21 @@ jobs:
     owner: psss
     project: tmt
 
+  # Release to copr
+  - job: copr_build
+    trigger: release
+    branch: main
```

We don't support `branch` for the `release` trigger (yet). Is this really needed? For now, you would need to make sure that you don't have the config in the branch you don't want to create releases from.

```suggestion
    trigger: release
```